services/account-without-juggler/repositories/account/datasources/mysql.json

```diff
@@ -0,0 +1,7 @@
+{
+    "host":"localhost",
+    "user":"root",
+    "password": "dis-mysql",
+    "database": "test-db",
```

Comment:
let's make it simpler for future users

password: "password" // password
database: "loopback-next-example" // name of example

should be consistent across all examples